.github/workflows/test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ['3.8', '3.9', '3.10', '3.11']
+        python: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
[while I was there] Added Python 3.12 to the versions we're testing :slightly_smiling_face: